### ISSUE TITLE
Does not detect libcxx LLVM

### ISSUE THREAD

**Issue by AnAverageHuman** [intel/intel-graphics-compiler#140]
The snippet below breaks when LLVM is compiled with libc++ instead of libstdc++:

https://github.com/intel/intel-graphics-compiler/blob/931ea39dedfc6cf536c9fc05a0db9df4943f86d3/IGC/CMakeLists.txt#L3278-L3288

```sh
$ llvm-config --libs
-lLLVM-10libcxx
```

**Comment by aus-intel** [comment]
Presumably should be fixed with https://github.com/intel/intel-graphics-compiler/commit/366754659acf09dc768afa8887e04889d5087494.

**Comment by AnAverageHuman** [comment]
Can confirm that IGC builds properly with 366754659acf09dc768afa8887e04889d5087494 applied, thanks!
